packages/react-cli/src/commands/fetch-configs.ts

```diff
@@ -0,0 +1,29 @@
+import { Command } from '@oclif/command';
+import { ensureDependencies } from '../tasks/ensure-dependiencies';
+import Listr from 'listr';
+import { downloadTSConfigFiles } from '../tasks/download-ts-config';
+import { exec } from '../utils/shell';
+
+export class FetchConfigsCommand extends Command {
+  static description = 'Fetches config files for linting and typescript projects';
+
+  static examples = ['$ react fetch-configs'];
+
+  async run() {
+    await ensureDependencies();
+
+    let path = './';
+    let tasks = new Listr([
+      {
+        title: 'Downloading shared config for DeveloperTown',
+        task: () => downloadTSConfigFiles(path),
```

Comment:
wondering if `downloadTSConfigFiles` should have a default argument which is equal to `./` so that we can just call `downloadTSConfigFiles()`.  This would largely depend on what path is doing here

Reply:
I think long term, this is going to be converted to an addon, so the command will actually be like:
```
react install @developertown/typescript-config-files
react g typescript-config-files
# ^ this command will either complain that the command doesn't exist if you execute it outside of a project
```

Reply:
documentation and stuff added. hopefully it's clearer :)